Create Asset Risk Assessment - bLUSD.md

DiligentDeer opened the pull request:
Risk assessment article on Liquity - Chicken Bonds for review.

benber86 commented on articles/liquity/Asset Risk Assessment - bLUSD.md:
```diff
@@ -0,0 +1,203 @@
+
+*[Liquity: Decentralized Borrowing Platform](https://docsend.com/view/bwiczmy) (Whitepaper)*
+
+Liquity is a decentralized borrowing protocol that allows interest-free loans against ETH as collateral. Loans are paid out in LUSD and need to maintain a minimum collateral ratio (CR) of 110%. The loans are secured by a Stability Pool containing LUSD that caters the liquidations.
```
change "caters" to "handles"

benber86 commented on articles/liquity/Asset Risk Assessment - bLUSD.md:
```diff
@@ -0,0 +1,203 @@
+
+Liquidation rewards are evenly distributed to the LUSD deposited in the Stability Pool. When the ETH price drops quickly, heavy liquidations might get triggered. If people are not confident with exposure to ETH, they might not deposit their LUSD in the stability pool. When the stability pool is empty, the debt obligation is transferred to other LUSD debt holders, reducing their collateral ratio.  ...
+
+In an undesirable scenario where the Stability Pool has no funds to facilitate the liquidations, the debt obligation from a Trove that is deemed to be liquidated might get transferred to other LUSD debt holders. Transferring this debt obligation will reduce the health factor (collateral ratio) of the remaining Troves making them vulnerable to liquidation if the ETH price chooses to go down.
```
"chooses to go down" -> "goes down"
(or "in the event of an adverse ETH price movement")

benber86 commented on articles/liquity/Asset Risk Assessment - bLUSD.md:
```diff
@@ -0,0 +1,203 @@
+>
+> **Liquity Whitepaper**
+
+The above ensures that when risky debt is redistributed to the existing debt holders, Recovery Mode is triggered. Recovery Mode is activated when the protocol's Total Collateral Ratio (TCR) falls below 150%. In this undesirable mode of operation, Troves with a collateral ratio between the current TCR and 110% also become subject to liquidation.
```
"ensures that Recovery Mode is triggered when risky debt..."

benber86 commented on articles/liquity/Asset Risk Assessment - bLUSD.md:
```diff
@@ -0,0 +1,203 @@
+
+Chicken Bonds are a novel mechanism for projects to bootstrap protocol-owned liquidity at no cost while boosting yield opportunities for end users. The amplified value accrual to bTKN is achieved by investing the funds under protocol-owned liquidity and the bonds’ (underlying tokens (TKN)).  
+
+Users may deposit TKN in exchange for an accruing balance of bTKN. At any time, bond-holders can either retrieve their principal foregoing the accrued amount (“chicken out”) or trade it in for the accrued bTKN (“chicken in”). Here, a portion of the TKN acquired by the system backs the bTKN supply, and this portion depends on the time of chicken_in.
```
The piece alternates between "chicken in" "`chicken_in`" and "CI", I'd suggest making it consistent. For readability just using "chicken in" as a regular verb seems better to me.
For instance, in the last part of this sentence: "... this portion depends on the time at which the bond holder chickened in"

DiligentDeer replied:
CI, Chicken_in, and chicken_in were changed to chicken in

benber86 commented on articles/liquity/Asset Risk Assessment - bLUSD.md:
```diff
@@ -0,0 +1,203 @@
+
+## A consequence of lowering the alpha
+
+A major reason for greater bond age is people not willing to take an action (CI or CO), in other words, people are not convinced with the currently accrued bLUSD is profitable enough to chicken_in. When the [current weighted average bond age](https://dune.com/queries/1460655) is greater than the target age which is 15 days, the controller would lower the alpha.
```
"not convinced with" => "not convinced that"

DiligentDeer replied:
Does this sound/read well?

A major reason for greater bond age is people not willing to take an action (Chicken in or Chicken out), in other words, people not convinced enough that the bLUSD accrued so far is profitable enough to chicken in. When the [current weighted average bond age](https://dune.com/queries/1460655) is greater than the target age which is 15 days, the controller would lower the alpha.

benber86 replied:
It sounds fine, though you give a reason for why people would refrain from chickening in but not for why they wouldn't chicken out?

DiligentDeer replied:
I just wanted to lead the readers to make them understand in what condition will the controller be triggered.

> why people wouldn't chicken out?

A user might not choose to chicken out if they feel the bLUSD market price can go up or maybe due to sunk cost fallacy. They might chicken out as well; it will depend on the market condition, their bond age etc.

I believe the above would not add much to what I want to convey (the impact of lowering the accrual parameter) so I didn't include it.

benber86 commented on articles/liquity/Asset Risk Assessment - bLUSD.md:
```diff
@@ -0,0 +1,203 @@
+Lowering the alpha will change the non-profitable chicken_in trade to a profitable chicken_in trade by accelerating the bLUSD accrual. If we were to create a bond for 1000 LUSD on Nov 2nd the break-even time (value of accrued bLUSD = value of LUSD bonded) considering the market price of bLUSD as $1.20 turn out on Nov 25th.
+![Demonstartion of expected bond age with breeak even point and rebonding time with 1000 LUSD as on November 2nd 2022](https://lh6.googleusercontent.com/wl-7yMaLz2OLSAYuQ2vmZCFnKliQOPv5YHwXogI61kj4UM_WeXuy1rJ78K7L8c-J3fVKivEiMUvOe1mReWhDtoNG_uK5Q8tmbg-seV1gIxVvJq5CrRZr0z6mO8-vqRWUZUa7EP8eKtqgNoudnHDrUrcbKxKp7b68CPOlLNWehQ_6iNOlWbW2jl3SYziFXw)
+
+Considering a zero-sum game, if there is only 1 user currently bonding with a considerable amount of LUSD. When his/her non-profitable trade is converted to a profitable one and when he/she chicken_in, the extra profit that he/she would gain will come from the bLUSD market price.
```
use "they,their,them" as gender neutral pronoun rather than "he/she,his/her,him/her".

DiligentDeer replied:
> if there is only 1 user currently bonding with a considerable amount of LUSD

I am referring to a single person here so using "they", "them", "their" will not be grammatically correct as these are plural and we need a singular pronoun which I am already using.

benber86 replied:
"They" can be used in the singular to refer to somebody of indefinite gender.
https://en.wikipedia.org/wiki/Singular_they#Contemporary_usage
It makes for less wordy writing than using he/she, him/her...

DiligentDeer replied:
Wow...I didn't know this. I will make the following change,

From:

> Considering a zero-sum game, if there is only 1 user currently bonding with a considerable amount of LUSD. When his/her non-profitable trade is converted to a profitable one and when he/she chicken_in, the extra profit that he/she would gain will come from the bLUSD market price.

To:

> Considering a zero-sum game, if there is only 1 user currently bonding with a considerable amount of LUSD, their non-profitable trade will be converted to a profitable one (based on the mechanic discussed above). When they chicken in, the extra profit (which otherwise would not be generated if $\alpha$ was not to be lowered) will come from the bLUSD inflation.

benber86 commented on articles/liquity/Asset Risk Assessment - bLUSD.md:
```diff
@@ -0,0 +1,203 @@
+
+### The risk associated with the yield-generating protocols
+
+bLUSD is exposed to the risks at B.Protocol and Yearn used for generating yield. Loss of funds from these protocol/vaults for any reason can severely impact the bLUSD floor price which is the biggest demand driver for holding blUSD or bonding LUSD.
```
I think there could be a bit more here on the risk from the associated protocols.
Is B.protocol audited? How long have they been around?
How long has the yearn strategy they rely on been around?

bout3fiddy commented on articles/liquity/Asset Risk Assessment - bLUSD.md:
```diff
@@ -0,0 +1,203 @@
+
+*[Liquity: Decentralized Borrowing Platform](https://docsend.com/view/bwiczmy) (Whitepaper)*
+
+Liquity is a decentralized borrowing protocol that allows interest-free loans against ETH as collateral. Loans are paid out in LUSD and need to maintain a minimum collateral ratio (CR) of 110%. The loans are secured by a Stability Pool containing LUSD that caters the liquidations.
```
`Loans are paid out in LUSD and need to maintain and min CR`: please rephrase sentence. Who needs to maintain the CR: loan or the debtor?

Finally: some indication here is needed. is the CR fixed? can it be changed? Do users pay interest upon minting or is it truly a free to use protocol where the user pays absolutely nothing, mints stablecoins, pays no interest?

DiligentDeer replied:
The following things were made clear (lines 13-18)
- Who needs to maintain the CR: the loan or the debtor?
- Is the CR fixed?
- Borrow and redemption fee.
- Maximum expected loss on immediate liquidation of a trove.

bout3fiddy commented:
One thing I miss here is a short discussion on the history of LUSD's peg. Nothing fancy, perhaps a graph showing Curve pool balances, and maybe a short description of how the LUSD peg mechanics work.

Our friend MonetSupply has always been a strong critic of Liquity, so perhaps some research explaining the pain points that Liquity faces is relevant here. For reference:

https://twitter.com/MonetSupply/status/1565932285890744325?s=20&t=Kzre4UuBimsF9WMa8-E-5w

If you want to go deeper into Monet's thoughts: https://twitter.com/search?lang=en&q=(liquity%20OR%20lusd)%20(from%3Amonetsupply)&src=typed_query

DiligentDeer commented:
> One thing I miss here is a short discussion on the history of LUSD's peg. Nothing fancy, perhaps a graph showing Curve pool balances, and maybe a short description of how the LUSD peg mechanics work.

I added 2 images and some information about LUSD peg stability.

> Our friend MonetSupply has always been a strong critic of Liquity, so perhaps some research explaining the pain points that Liquity faces is relevant here. For reference:
> https://twitter.com/MonetSupply/status/1565932285890744325?s=20&t=Kzre4UuBimsF9WMa8-E-5w

 Tweet:
> risk of trove being redeemed if it is low c-ratio

Redemption = repayment and repayment is carried out at a face value i.e. 100% CR. If a debt is fully prepared due to the redemption of LUSD the debt holder will get the remaining collateral back.

> risk of user needing to repay debt to avoid liquidation, effectively forcing them into shorter holding period/paying fee multiple times

I tried simulating a debt position in defi saver, there are no extra fees other than gas for repayment of the debt or even adding the ETH to manage the CR.

DiligentDeer commented:
Added this new file: **bLUSD Risk Assessment thread.md**
- Tried to make it short and simple.

Updated the following file: **Asset Risk Assessment - bLUSD.md**
- Cut down unwanted content and refined it with Grammarly.

I request a review again if possible @bout3fiddy @benber86
